src/memphis.ts

```diff
@@ -349,9 +349,9 @@ class Memphis {
       let schemaUpdateSubscription =
         this.schemaUpdatesSubs.has(internalStationName);
       if (schemaUpdateSubscription) {
-        this.producersPerStation.set(
+        this.clientsPerStation.set(
           internalStationName,
-          this.producersPerStation.get(internalStationName) + 1
+          this.clientsPerStation.get(internalStationName) + 1
```

Comment:
on line 358 we should add  if(schemaUpdateData && ...) for cases of users that upgraded their SDK and didn't upgrade the broker.
Please check if it requires a fix on the other SDKs as well